### PR TITLE
Main page mobile

### LINKED ISSUE #10: (3ptx) - Main Page Mobile
Create Main page for mobile.

### PR DESCRIPTION
**Pull request by gjuliao**
- Create the main view with dummy data.
- No linter errors

You can review below of image of the mobile suggested version:
image

### CODE REVIEW

File: src/components/Main.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import bmw from '../assets/images/bmw.png';
```
Comment (Juank628) [inline src/components/Main.jsx:2]: Please, try to compress the image. 1.3MB is too heavy.
You can use this page to compress the image: https://tinypng.com/

**Review by Juank628 — CHANGES_REQUESTED** [review]
Please, try to compress the BMW image. 1.3MB is too heavy.
You can use this page to compress the image: https://tinypng.com/

File: src/components/Main.jsx (hunk 1 above)
Comment (gjuliao) [reply src/components/Main.jsx:2]: Hi Juan, is has been edited already.

Looking forward for your approval.

Thanks!

Comment (Juank628) [reply src/components/Main.jsx:2]: Thanks! approving now!

**Review by Juank628 — APPROVED** [review]
Excellent!!! It's time to merge!

**Pull request merged**